spec/factories/tasks.rb

```diff
@@ -5,5 +5,10 @@
     from = Date.parse("2019/08/01")
     to   = Date.parse("2019/12/31")
     deadline { Random.rand(from..to) }
+
+    #ステータスが完了しているタスク
+    trait :status_done do
+      status { (2) } 
```

Comment:
数字で記載しても何を示しているか分からないので、`enum`はSymbol形式で記載するようにしましょう。
```suggestion
      status { :done } 
```

Reply:
a5f4de7にて変更いたしました。

Reply:
a5f4de7 にて変更いたしました